src/Language/PureScript/Interactive.hs

```diff
@@ -217,12 +226,15 @@ handleImport
   => ImportedModule
   -> m ()
 handleImport im = do
-   st <- gets (updateImportedModules (im :))
+   st <- gets (updateImportedModules (renew im))
    let m = createTemporaryModuleForImports st
    e <- liftIO . runMake $ rebuild (map snd (psciLoadedExterns st)) m
    case e of
      Left errs -> printErrors errs
      Right _  -> put st
+   where
+   renew :: ImportedModule -> [ImportedModule] -> [ImportedModule]
```

Comment:
Yes, that's what I fixed with the patch cd13ee4234203e9faed27c488ca55b7edc8e9e49 included in this PR.

Before:

```
> import Data.Functor
> import Data.Functor
> import Data.Functor
> import Data.Functor
> :show import
import Data.Functor
import Data.Functor
import Data.Functor
import Data.Functor

>
```

After:

```
> import Data.Functor
> import Data.Functor
> import Data.Functor
> import Data.Functor
> :show import
import Data.Functor

>
```